spec/feature/index_post_view_spec.rb

```diff
@@ -0,0 +1,97 @@
+      comments_counter: 1,
+      likes_counter: 2
+    )
+
+    @post2 = Post.create(
+      author: @user,
+      title: 'Algorithm',
+      text: ' An algorithm is a set of instructions for solving a problem or accomplishing a task. One commonexample of an algorithm is a recipe, which consists of specific instructions for preparing a dish or meal. a process or set of rules to be followed in calculations or other problem-solving operations,especially by a computer.',
+      comments_counter: 1,
+      likes_counter: 2
+    )
+
+    @post3 = Post.create(
+      author: @user,
+      title: 'What is a web designer do?',
+      text: 'Web designers plan, create and code internet sites and web pages, many of which combine text with sounds, pictures, graphics and video clips. A web designer is responsible for creating the design and layout of a website or web pages. It and can mean working on a brand new website or updating an already existing site.',
+      comments_counter: 0,
+      likes_counter: 1
+    )
+
+    @comment = Comment.create(
+      post: @post1,
+      author: @user,
+      text: 'This post really gives ideas what women role in tech.'
+    )
+  end
+
+  before { subject.save }
+
+  describe 'testing for user post index page', js: true do
+    it 'displays the profile picture for each user.' do
+      visit user_posts_path('1')
+      page.has_content?('Screenshot Image')
+      expect(page).to have_css('img')
+    end
+
+    it 'displays the user profile picture.' do
+      visit user_posts_path('1')
+      expect(page).to have_content(@user.name)
+    end
+
+    it 'display the number of posts each user has written.' do
+      visit user_posts_path('1')
+      page.has_content?("Number of posts: #{@user.posts_counter}")
+    end
+
+    it 'display the title of the post' do
+      visit user_posts_path('1')
+      expect(page).to have_content(@post1.title.to_s)
+    end
+
+    it 'I can see some of the post body' do
+      visit user_posts_path('1')
+      expect(page).to have_content(@post1.text.to_s)
+    end
+
+    it 'I can see the first comments on a post' do
+      visit user_posts_path('1')
+      expect(page).to have_content('Title: What is the role of women in technology?')
+    end
+
+    it 'I can see how many comments a post has.' do
+      visit user_posts_path('1')
+      page.has_content?("Comments counter: #{@post1.comments_counter}")
+    end
+
+    it 'I can see how many likes a post has.' do
+      visit user_posts_path('1')
+      page.has_content?("Likes counter: #{@post1.likes_counter}")
+    end
+
+    it 'I can see a section for pagination if there are more posts than fit on the view.' do
+      visit('/users/1/posts')
+      page.has_button?('Pagination')
+    end
+
+    it 'When I click on a post, it redirects me to that post show page.' do
+      visit user_posts_path('1')
+      click_on(@post1.title)
+      expect(page).to have_current_path user_post_path(@post1.author_id, @post1)
+    end
+  end
+end
```

Comment:
- [ ] Just as you were told by the previous reviewer, When I run the unit tests they fail. The error being shown is ```An error occurred while loading ./spec/feature/index_post_view_spec.rb.```

![image](https://user-images.githubusercontent.com/86318284/208915188-ae503264-eb7c-406a-b542-d59aa098d431.png)